datasets/labelme/converter.py

```diff
@@ -0,0 +1,125 @@
+# %%
+import json
+import sys
+from pathlib import Path
+import os
+import numpy as np
+
+# %%
+if len(sys.argv) != 3:
+    sourcePath = (
+        "/home/dubar/Bureau/peerannot/fork/peerannot/datasets/labelme/data/LabelMe/"
+    )
+    targetPath = (
+        "/home/dubar/Bureau/peerannot/fork/peerannot/datasets/labelme/Testsyslink/"
+    )
```

Comment:
This is a bit weird to have local path hard coded in the packages, no (same issue as above).